fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java

```diff
@@ -166,7 +179,12 @@ protected void runAfterCatalogReady() {
                 // the heartbeat rpc's timeout is 5 seconds, so we will not be blocked here too long.
                 HeartbeatResponse response = future.get();
                 if (response.getStatus() != HbStatus.OK) {
-                    LOG.warn("get bad heartbeat response: {}", response);
+                    if ((response.getType() != HeartbeatResponse.Type.BACKEND ||
```

Comment:
shall switch the condition, could be more intuitive for human thinking.

Reply:
Good suggestion.